src/mlpack/core/boost_backport/CMakeLists.txt

```diff
@@ -1,12 +1,6 @@
 # Define the files that we need to compile.
 # Anything not in this list will not be compiled into mlpack.
 set(SOURCES
-  bernoulli.hpp
-  detail/bernoulli_details.hpp
-  detail/polygamma.hpp
-  detail/unchecked_bernoulli.hpp
-  math_fwd.hpp
-  policy.hpp
```

Comment:
After #3006 and this are merged, we can remove this directory entirely, I think! :tada: